keyboards/amptrics/0422/custom_led.c

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2024 Vinod Chowl (@vchowl)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2.0 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "gpio.h"
+
+#define LED_LAYER_0 B3
+#define LED_LAYER_1 B4
+#define LED_LAYER_2 B5
+#define LED_LAYER_3 B6
+
+#define LED_PINS_COUNT 4
+
+long unsigned int pins[LED_PINS_COUNT] = {LED_LAYER_0, LED_LAYER_1, LED_LAYER_2, LED_LAYER_3};
+
+void update_leds_for_layer(uint8_t layer) {
+    for (int i = 0; i < LED_PINS_COUNT; i++) {
+        gpio_write_pin_low(pins[i]); // Turn off all LEDs
+    }
```

Comment:
Instead of having repeated code, you could just call this function here:
```suggestion
    turn_off_all_leds();
```